sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -0,0 +1,200 @@
+    // (undocumented)
+    platform: "apple";
+}
+
+// @public
+export interface BaiduInstallation extends DeviceTokenInstallation {
+    // (undocumented)
+    platform: "baidu";
+}
+
+// @public
+export interface BaiduMessage extends NotificationHubMessage {
+    // (undocumented)
+    contentType: "application/json;charset=utf-8";
+    // (undocumented)
+    platform: "baidu";
+}
+
+// @public
+export interface BrowserInstallation extends Installation {
+    // (undocumented)
+    platform: "browser";
+    // (undocumented)
+    pushChannel: BrowserPushChannel;
+}
+
+// @public
+export interface BrowserMessage extends NotificationHubMessage {
+    // (undocumented)
+    contentType: "application/json;charset=utf-8";
+    // (undocumented)
+    platform: "browser";
+}
+
+// @public
+export interface BrowserPushChannel {
+    auth: string;
+    endpoint: string;
+    p256dh: string;
+}
+
+// @public
+export interface DeviceTokenInstallation extends Installation {
+    // (undocumented)
+    pushChannel: string;
+}
+
+// @public
+export interface FirebaseLegacyInstallation extends DeviceTokenInstallation {
+    // (undocumented)
+    platform: "gcm";
+}
+
+// @public
+export interface FirebaseLegacyMessage extends NotificationHubMessage {
+    // (undocumented)
+    contentType: "application/json;charset=utf-8";
+    // (undocumented)
+    platform: "gcm";
+}
+
+// @public
+export interface Installation {
+    readonly expirationTime: string;
+    installationId: string;
+    readonly lastUpdate: string;
+    platform: string;
+    tags?: string[];
+    templates?: Record<string, InstallationTemplate>;
+    userId?: string;
+}
+
+// @public
+export interface InstallationPatch {
+    op: JSONPatchType;
+    path: string;
+    value?: string;
+}
+
+// @public
+export interface InstallationTemplate {
+    body: string;
+    expiry?: string;
+    headers: Record<string, string>;
+    tags?: string[];
+}
+
+// @public
+export type InstallationType = AppleInstallation | ADMInstallation | BaiduInstallation | BrowserInstallation | FirebaseLegacyInstallation | WindowsInstallation;
+
+// @public
+export type JSONPatchType = "add" | "remove" | "replace";
+
+// @public
+export interface NotificationHubMessage {
+    body: string;
+    contentType: string;
+    headers: Record<string, string>;
+    platform: string;
+}
+
+// @public
+export type NotificationHubMessageType = AppleMessage | ADMMessage | BaiduMessage | BrowserMessage | FirebaseLegacyMessage | WindowsMessage | TemplateMessage;
+
+// @public
+export interface NotificationHubResponse {
+    correlationId?: string;
+    location?: string;
+    trackingId?: string;
+}
+
+// @public
+export class NotificationHubsClient extends ServiceClient {
+    constructor(connectionString: string, hubName: string, options?: NotificationHubsClientOptions);
+    deleteInstallation(installationId: string, options?: OperationOptions): Promise<NotificationHubResponse>;
```

Comment:
Is this a long-running operation? if so, did you consider returning a poller for it?

Reply:
Same comment about the other operations.

Reply:
No, none of these are LROs, instead they are async operations that happen in a queue.